app/services/PartialPayExtractor.scala

```diff
@@ -37,15 +37,9 @@ trait PartialPayExtractor extends PeriodHelper with FurloughPeriodExtractor {
       }
     }
 
-  def getPeriodRemainder(partialPeriod: PartialPeriod): Period = {
-    val original = partialPeriod.original
-    val partial = partialPeriod.partial
-
-    if (partial.start.isAfter(original.start)) {
```

Comment:
this fails when we the furlough falls in only one pay period ,  in that case this `if` is always true and even called for After Furlough pages the before furlough dates will be shown